sam_view.c

```diff
@@ -817,6 +825,37 @@ static int getLength(char **s)
     return n;
 }
 
+void copy_tag(const char *tag, const bam1_t *rec, kstring_t *linebuf)
+{
+    uint8_t *s = bam_aux_get(rec, tag);
+    if (s) {
+        char aux_type = *s;
+        switch (aux_type) {
+            case 'C':
+            case 'I':
+            case 'S': aux_type = 'I'; break;
+            case 'c':
+            case 's': aux_type = 'i'; break;
```

Comment:
Initially I thought there was a bug as that you have C/I/S above and only c/s below it, but I see the I->I is just a nop.  I'll nuke the ```case 'I'``` line on merge unless you have an objection.